examples/AI-surrogate-mode-hard-magnets.ipynb

```diff
@@ -0,0 +1,384 @@
5w1axh3p5S70etx/bu1hKjK3Yc4FKhtAHAAAAAOAy3ZvWLNzX59j5ij+4ywOSf/UrBkxp3YyKPxeoRAh9AAAAAAAuExLgqw4F+vo4ZImXf4jUdZT92JZ5UtrFij8bqCQIfQAAAAAALlVw6/Z1sYmOeXCvcZJxxdfejEvS5jmOeTZQCRD6AAAAAABcKrK5fTNnh/T1kaQaTaR2t+cf1+8shbWq+HOBSsLH1QUAAAAAADxb92Y15WVI2WbOcYYtW1uOndd1rWpX/OFRT0nZWVLUk1KTKMkwSr4HqCJ40wcAAAAA4FIhAb7q6Iy+PpLUuId03/+kptcR+MDjEPoAAAAAAFwusoX9Eq+Yww7q6wN4MEIfAAAAAIDL9SrQzHnrsQtKy3RAXx/AgxH6AAAAAABcrnuzWvK6YvVVhi1bm4+dd11BQBVA6AMAAAAAcLnqAb7qUKivj5OWeJmmdORXadsC5zwfcBPs3gUAAAAAcAuRLcK0Pf5i3rHDmjnnysqQdn0hxbwjndwmBYRKbYdI/sGOnQdwE7zpAwAAAABwC5EF+/rEObivz8U46cvf5wQ+kpR2Udr2qeOeD7gZQh8AAAAAgFso1NcnK1tbjl1w3ARhLaWIW+zHYt6VsrMdNwfgRgh9AAAAAABuoXqAr9o3KNjXx8FLvKKesD9OPCQd+MGxcwBugtAHAAAAAOA2Ci7xcnjo06yPFN7Rfiz6v46dA3AThD4AAAAAALcR2SLM7niLo/v6GIYU+bj92JHV0sntjpsDcBOEPgAAAAAAt9G9WS0ZBfr6bI274NhJOt4jBdW1H4t517FzAG6A0AcAAAAA4DZCq/mqfYPqdmMOX+Ll4y/1eMR+bOfnUnKCY+cBXIzQBwAAAADgViKb2y/xcnjoI0ndH5a8/fOPbRnShg8cPw/gQoQ+AAAAAAC3UqivzzEH9/WRpOA60rXD7cc2fihlXnbsPIALEfoAAAAAANxKj+b2fX3Ss7K1zdF9fSQpssD27annpO0LHT8P4CKEPgAAAAAAtxJazVfX1C/Y1yfR8ROFXyO16G8/FvOuZJqOnwtwAUIfAAAAAIDbKbjEyyl9fSQp6kn74zN7pEM/O2cuwGKEPgAAAAAAt1Mw9Nl87LzSsxzc10eSWt4o1Y7IP65WU7p02vHzAC5A6AMAAAAAcDs9mxXV1+ei4yfy8pIiH88Jfm6bLk3YLXW+3/HzAC5A6AMAAAAAcDuhgb5qV69gXx8nLfHqMkp6Yl3ONu5+gc6ZA3ABQh8AAAAAgFuyrK+Pt0/OGz9AFcPvagAAAACAW4psUcvueNNRJ/X1AaooQh8AAAAAgFvq2bxwX5/t8U7o6wNUUYQ+AAAAAAC3VCPQT20L9vU55KQlXpJ0NNr+OOmkdHJb2e8D3AShDwAAAADAbRVc4hVz2Emhz4p/SrNukdZMl05slb54THqjo/T1U5JpFn/fmuk59634p3PqAiqA0AcAAAAA4LYKNnPedPS8MrKyHTvJ0Whp1dScz8umSDP7SdsXSNmZ0qnt0tG1Rd+3ZnrO9VLO/bzxAzdD6AMAAAAAcFu9CvT1ScvM1vb4C46dpGmUdNOU4s/HvFN47MrAR8q5v2mUY+sCKojQBwAAAADgtmoE+qlNeIjdmFO2bu89ofjgZ+9iKTE2/7iowKf3BMfXBFQQoQ8AAAAAwK0VXOIVE5vonImKDX5Mad2MnI8EPqhECH0AAAAAAG6tYOiz8Wii4/v65Cou+NkyT1r5TwIfVCqEPgAAAAAAt9aruf0OXk7p63Ol3hOk68fbj2VcklZOzT8m8EElQOgDAAAAAHBrNYP81LaefV+fdYedtMQr181/l+peU/Q5Ah9UEoQ+AAAAAAC3V7ivjxOaORc0+NXCYwGhBD6oNAh9AAAAAABuL7KF/RKvjUfOO6+vT664DYXH0i7mNHMGKgFCHwAAAACA2+vZ3P5Nn8uZNu04fsF5E66ZLv38j6LPLZtC8INKgdAHAAAAAOD2ahXR18dpW7cX3Ja9KAQ/qAQIfQAAAAAAlULBXbyc0tenYODTf7LkE1D0tQQ/cHOEPgAAAACASqFgM+eNR84r0+bAvj4FA5+bpkj9/iS1uMH+uit39SL4gRsj9AEAAAAAVAo9C7zpcznTpu3xFx3z8KICn9xdutrcmvOfweFSt4ekQf+Xcz4XwQ/clI+rCwAAAAAAoDTCgv3VJjxE+xKS88ZiYs+pW9OaFXvw0ejiAx9Jajc05+2eht0kr9/enWjZP+c/c+9bNkVqHCk1japYLYAD8aYPAAAAAKDS6FVg6/Z1hx3QzLlplNTv+ZzPBQMfSQqsJTXukR/45Oo9If+Nn37PE/jA7fCmDwAAAACg0ohsEaY50UfzjjceSVSmLVu+3hV8p6H/n3N695Q1uOk9gTd84LZ40wcAAAAAUGkU7OuTmmHTjuMO6utT3uCGwAduitAHAAAAAFBp1A72V0R4sN2YU7ZuB6oAQh8AAAAAQKVScOv2dbEO6OtTVpfPS3u+s35eoAwIfQAAAAAAlUqv5vahT25fH6fLSpfWzZA+vl36dytpwQPSuUPOnxcoJ0IfAAAAAEClUnAHr5QMm3Y6qq/P1Xj5Sr+8Jh1eJWVn5YztXez8eYFyIvQBAAAAAFQqtYP91bpuwb4+Fizx8vKS2gy2HyP0gRsj9AEAAAAAVDqF+voctqiZc9sh9sdx66RLZ6yZGygjQh8AAAAAQKVTcInXhsOJyrKir0/zfpJv0BUDprR/qfPnBcqB0AcAAAAAUOkUbOackmHTzhNJzp/YN0BqdaP9GEu84KYIfQAAAAAAlU6dEH+1KtTXx0VLvGJXSBkp1swNlAGhDwAAAACgUoossMTLstCn9UDJ8M4/zkqTDq2wZm6gDAh9AAAAAACVUsFmzhuPnLemr09gLanpdfZjLPGCGyL0AQAAAABUSj2b27/pcyk9S7us6OsjFV7itX+pZMuyZm6glAh9AAAAAACVUt2QALWsE2Q3ZtkSrza32h9
```

Comment:
I think we should add some more text explaining cells 9 and 10. For cell 10 e.g.: what is going on > 1000K?